### PR TITLE
UI show event detail

### PR DESCRIPTION
**Pull request by limkimhuor**

### CODE REVIEW

File: app/views/events/show.html.erb [hunk 1]
```diff
@@ -1,27 +1,68 @@
...
+        <div class="ev-att">
+          <div class="contain">
+            <div class="lb-attendees">
+              <%= label_tag :lattendee, t("events.labels.attendees") %>
+            </div>
+            <div class="add-att">
+              <%= form_for @attendee, remote: true do |f| %>
+                <%= f.hidden_field :event_id, value: @event.id %>
+                <div class="form-field">
+                  <%= f.select :user_id, @users.collect{ |a| [a.name, a.id] }, {},
```
Comment (tqtrung09) [inline app/views/events/show.html.erb]: no space after {

File: app/views/events/show.html.erb [hunk 2]
```diff
@@ -1,27 +1,68 @@
...
-      remote: true, class: "delete_attendee" %>
-    </li>
-  <% end %>
+<div id="maincell">
+  <div class="contain">
+    <div class="form-action">
+      <%= link_to root_path, title: t("events.buttons.go_back"),
+        "data-no-turbolink": true, class: "btn-go-back" do %>
+          <i class="go-back"></i>
+        <% end %>
```
Comment (vietbq) [inline app/views/events/show.html.erb]: convention

**Comment by daodat18plus** [comment]
OK

File: app/models/event.rb [hunk 3]
```diff
@@ -38,6 +38,10 @@ def json_data
     }
   end
 
+  def not_same_day
```
Comment (dieunb) [inline app/models/event.rb]: bad naming

File: app/views/events/show.html.erb [hunk 4]
```diff
@@ -1,27 +1,58 @@
...
+                </li>
+              <% end %>
+            </ul>
+          </div>
+        </div>
+        <div class="lb-contain">
+          <div class="lbcal">
+            <ul>
+              <li><%= label_tag :lbcal, t("events.labels.calendar") %></li>
+              <li><%= @event.calendar.name %></li>
```
Comment (dieunb) [inline app/views/events/show.html.erb]: Dùng delegate đi em

File: app/models/event.rb [hunk 5]
```diff
@@ -38,6 +39,10 @@ def json_data
     }
   end
 
+  def compare_date
```
Comment (dieunb) [inline app/models/event.rb]: is_diff_between_start_and_finish_date?

**Pull request merged**